### ISSUE TITLE
Possible false positive for comments within a run command.

### ISSUE THREAD

**Issue by deepio** [RedCoolBeans/dockerlint#54]
### Issue details
I'm getting an `Empty / bogus instruction is invalid` but i'm pretty sure its ok. 

### Steps to reproduce/test case

This is part of my dockerfile. This runs correctly in docker. 

```
RUN apk update \
  # psycopg2 dependencies
  && apk add --virtual build-deps gcc python3-dev musl-dev \
  && apk add postgresql-dev \
```

I have multiple lines after this one that start with `&&` and they do not have this error. 

### Affected version(s)

Latest, used by this repo <a href="https://github.com/AtomLinter/linter-docker">(AtomLinter/linter-docker)</a> which is a wrapper for redcoolbeans/dockerlint. <a href="https://github.com/AtomLinter/linter-docker/issues/56">Link to other issue.</>

**Comment by rnagy** [comment]
fixed by d775293312e3444510ba99a3856ffc25b14d94d7
